seahorn/trusty/user/base/lib/tipc/include/sea_tipc_helper.h

```diff
@@ -0,0 +1,8 @@
+#pragma once
+
+#include <trusty_ipc.h>
+
+
+
+ssize_t send_msg(handle_t handle, struct ipc_msg *msg);
+int msg_send_called;
```

Comment:
comment on what this global variable is for. 

By convention, we name globals using `g_` prefix, so it should be `g_msg_send_called`

Reply:
Not sure this variable should be declared in the header file.

Reply:
what is a better place to declare global variables?